docs/buckets.rst

```diff
@@ -0,0 +1,99 @@
+Buckets
+#######
+
+Buckets are a group of collections, shared between users, with fined
+permissions on the data stored inside.
+
+Basically a bucket have got an id and a list of people identifier that
+can administrate it.
+
+A bucket can be created using a PUT on the bucket URI:
+
+.. code-block:: http
+
+   > PUT /buckets/servicedenuages HTTP/1.1
+   < 201 Created
+
+    {
+      "id": "servicedenuages",
+      "owners": ["email:natim@example.com"]
+    }
+
+
+There are two kinds of data linked to a bucket:
+
+ - collections
+ - groups
+
+
+Collections
+===========
+
+Creating a collection inside a bucket enable all buckets owners to
+administrate the collection.
```

Comment:
Yes also since I didn't identify any permissions for bucket it does not means a lot of thing to me. Please elaborate on your proposal with permissions. I did not get what you wanted to do.